package.json

```diff
@@ -4,6 +4,7 @@
   "author": "Tobias Koppers @sokra",
   "description": "script loader module for webpack",
   "dependencies": {
+    "loader-utils": "~1.1.0",
```

Comment:
Not sure - it's supposed to work for webpack 2, but it doesn't specify the behavior for webpack 1: https://github.com/webpack/loader-utils